packages/react-urql/e2e-tests/useQuery.spec.tsx

```diff
@@ -2,7 +2,35 @@
 
 import * as React from 'react';
 import { mount } from '@cypress/react';
-import { Provider, createClient, gql, useQuery } from '../src';
+import {
+  Provider,
+  createClient,
+  gql,
+  useQuery,
+  dedupExchange,
+  cacheExchange,
+  fetchExchange,
+  Exchange,
+} from '../src';
+import { mergeMap, pipe, fromPromise } from 'wonka';
+
+const delayExchange: Exchange = ({ forward }) => {
+  return ops$ => {
+    return pipe(
+      ops$,
+      mergeMap(op => {
+        return fromPromise(
+          new Promise(res => {
+            setTimeout(() => {
+              res(op);
+            }, 250);
+          })
+        );
+      }),
+      forward
```

Comment:
We don't really want to delay results or teardown operations, but we're putting this exchange in front of `fetchExchange`, so we can exploit that and can likely just do:

```suggestion
    return pipe(
      ops$,
      forward,
      delay(250)
```